src/read_utils.rs

```diff
@@ -268,14 +267,12 @@ where
                 .flexible(true)
                 .trim(csv::Trim::All)
                 .from_reader(reader);
-            let mut objects: Vec<O> = vec![];
-            for object in rdr.deserialize() {
-                let object: O = skip_error_and_log!(
-                    object.with_context(|_| format!("Error reading {:?}", path)),
-                    LogLevel::Warn
-                );
-                objects.push(object);
-            }
+            use skip_error::SkipError;
+            let objects = rdr
+                .deserialize()
+                .map(|object| object.with_context(|_| format!("Error reading {:?}", path)))
+                .skip_error_and_log(LogLevel::Warn)
```

Comment:
wouln't warn be a better default value since you always use it with this value ?

Reply:
We do not have a "default value" mechanism for now, so I'm not sure what you mean by that. However, I thought we could also add some convenience methods in `skip_error`'s crate like `skip_error_and_warn()`, `skip_error_and_info()`, etc...

Reply:
of I though I'd seen a `log::Info` default value in skip_error 🤔